src/SFA.DAS.Commitments.Api/Orchestrators/EmployerOrchestrator.cs

```diff
@@ -215,5 +218,37 @@ public EmployerOrchestrator(IMediator mediator, ICommitmentsLogger logger)
                 UserId = userId
             });
         }
+
+        public async Task<GetPendingApprenticeshipUpdateResponse> GetPendingApprenticeshipUpdate(long accountId, long apprenticeshipId)
+        {
+            _logger.Info($"Getting pending update for apprenticeship {apprenticeshipId} for employer account {accountId}");
```

Comment:
But that would have to go in the request handler after the apprenticeship has been fetched from db.